### PR TITLE
SCB-210 remove extra useless config item

### PR DESCRIPTION
**Pull request by lijasonvip**
Signed-off-by: lijasonvip <libo75@huawei.com>

Follow this checklist to help us incorporate your contribution quickly and easily:

 - [x] Make sure there is a [JIRA issue](https://issues.apache.org/jira/browse/SCB) filed for the change (usually before you start working on it).  Trivial changes like typos do not require a JIRA issue.  Your pull request should address just this issue, without pulling in other changes.
 - [x] Each commit in the pull request should have a meaningful subject line and body.
 - [x] Format the pull request title like `[SCB-XXX] Fixes bug in ApproximateQuantiles`, where you replace `SCB-XXX` with the appropriate JIRA issue.
 - [x] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
 - [x] Run `mvn clean install` to make sure basic checks pass. A more thorough check will be performed on your pull request automatically.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

---
Use apollo.config.serverUri to note address of Apollo configuration.
Remove cse.config.client.serverUri which is useless.

### CODE REVIEW

File: foundations/foundation-config/src/main/java/org/apache/servicecomb/config/ConfigUtil.java [hunk 1]
```diff
@@ -150,7 +151,7 @@ private static void duplicateServiceCombConfigToCse(ConcurrentCompositeConfigura
   }
 
   public static DynamicWatchedConfiguration createConfigFromConfigCenter(Configuration localConfiguration) {
-    if (localConfiguration.getProperty(configCenterUrlKey) == null) {
+    if ((localConfiguration.getProperty(configCenterUrlKey) == null) && (localConfiguration.getProperties((configApolloUrlKey)) == null )) {
```
Comment (wujimin) [inline foundations/foundation-config/src/main/java/org/apache/servicecomb/config/ConfigUtil.java]: each time, we add a new IMPL.
we need to modify this place.
it's very bad.

Comment (lijasonvip) [reply foundations/foundation-config/src/main/java/org/apache/servicecomb/config/ConfigUtil.java]: OK, lets remove this check here and do the check after we get the configCenterSource.

File: foundations/foundation-config/src/main/java/org/apache/servicecomb/config/spi/ConfigCenterConfigurationSource.java [hunk 2]
```diff
@@ -22,5 +22,6 @@
 import com.netflix.config.WatchedConfigurationSource;
 
 public interface ConfigCenterConfigurationSource extends WatchedConfigurationSource {
+  boolean isValidSource(Configuration localConfiguration);
```
Comment (liubao68) [inline foundations/foundation-config/src/main/java/org/apache/servicecomb/config/spi/ConfigCenterConfigurationSource.java]: For compatibility, this method should give default implementation. But it's fine now , this is not known implementations.

File: dynamic-config/config-apollo/src/main/java/org/apache/servicecomb/config/archaius/sources/ApolloConfigurationSourceImpl.java [hunk 3]
```diff
@@ -49,11 +49,21 @@
...
+  private static final String APOLLO_CONFIG_URL_KEY = "apollo.config.serverUri";
+
   public ApolloConfigurationSourceImpl() {
   }
 
   private final UpdateHandler updateHandler = new UpdateHandler();
 
+  @Override
+  public boolean isValidSource(Configuration localConfiguration) {
+    if (localConfiguration.getProperty(APOLLO_CONFIG_URL_KEY) == null){
```
Comment (WillemJiang) [inline dynamic-config/config-apollo/src/main/java/org/apache/servicecomb/config/archaius/sources/ApolloConfigurationSourceImpl.java]: It's better to write the Log here (to let the user know about which configure is wrong).

Comment (lijasonvip) [reply dynamic-config/config-apollo/src/main/java/org/apache/servicecomb/config/archaius/sources/ApolloConfigurationSourceImpl.java]: Fixed.

**Pull request merged**